File: treebolicClientsIface/src/main/java/org/treebolic/clients/iface/IConnectionListener.java
/*
 * Copyright (c) 2023. Bernard Bou
 */

package org.treebolic.clients.iface;

/**
 * Model consumer interface
 *
 * @author Bernard Bou
 */
public interface IConnectionListener
{
	/**
	 * Connected callback
	 *
	 * @param success whether connection is successful
	 */
	void onConnected(@SuppressWarnings("SameParameterValue") boolean success);
}

File: treebolicClientsIface/src/main/java/org/treebolic/clients/iface/IConnectionListener.kt
/*
 * Copyright (c) 2023. Bernard Bou
 */
package org.treebolic.clients.iface

/**
 * Model consumer interface
 *
 * @author Bernard Bou
 */
interface IConnectionListener {

    /**
     * Connected callback
     *
     * @param success whether connection is successful
     */
    fun onConnected(success: Boolean)
}
